dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -276,6 +283,17 @@ def change_config(python, backend, cheatsheet, asciiart):
         console.print(f'[blue]Backend cached_value {backend}')
 
 
+@option_verbose
+@main.command(name='build-docs')
+@click.option('--docs-only', is_flag=True)
+@click.option('--spellcheck-only', is_flag=True)
+@click.option('--package-filter', type=click.Choice(get_available_packages()), multiple=True)
+def build_docs(verbose: bool, docs_only: bool, spellcheck_only: bool, package_filter: Tuple[str]):
+    """
+    Builds documentation in the container
+    """
+
```

Comment:
Yeah. The only thing missing here is to actually run the `./docs/build-docs.py` inside the breeze container.

Reply:
yes I will add it @potiuk I was not very sure if I have to invoke build_docs.py directly or via docker command. Now I am clear on that part.